platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -743,12 +773,12 @@ aziotedge_check() {
     base_message="IoT Edge requires a minimum storage space of $binary_size MB for installing edge daemon and $container_size MB for installing runtime docker containers. We verified that the the device has $available_storage MB of available storage for File System $(df -P -m "$MOUNTPOINT" | awk '{print $6}')"
 
     if [ $ret -eq 0 ]; then
-        wrap_warning "$base_message"
+        wrap_warning_message "$base_message"
         #TODO : Check with PM on messaging
-        wrap_warning "Additional storage space maybe required for based on usage of iotedge and has not been measured here.Please visit aka.ms/iotedge for more information"
+        wrap_warning_message "Additional storage space maybe required for based on usage of iotedge and has not been measured here.Please visit aka.ms/iotedge for more information"
     elif [ $ret -eq 1 ]; then
-        wrap_warning "$base_message"
-        wrap_warning "If you are planning to install iotedge on a different mountpoint, please run the script with MOUNTPOINT='<Path-to-mount>' $(basename "$0")"
+        wrap_warning_message "$base_message"
+        wrap_warning_message "If you are planning to install iotedge on a different mountpoint, please run the script with MOUNTPOINT='<Path-to-mount>' $(basename "$0")"
```

Comment:
Instead of asking a Cx to run a particular command, can we just have a prompt and let the Cx fill in the parameter? 
Unless we want it to be solely uninteractive, we have print out this log and they can set it themselves.

Reply:
The script is meant to run solely uninteractive, We want to keep the prompts to the minimum since this is a "check" and not "install" script.